packages/terra-clinical-result/src/terra-dev-site/test/clinical-result/clinical-result-blood-pressure/EnteredInErrorClinicalResultBloodPressureNoUnit.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ClinicalResultBloodPressure } from 'terra-clinical-result/lib/index';
+
+const resultData = {
+  systolic: {
+    eventId: '111.1',
+    result: {
+      value: '180',
+      unit: 'mmHg',
```

Comment:
Could you add a test where `unit` isn't present in the data?

Reply:
Changed tests around here, added two tests for unit not being present - one for blood pressure result and one for clinical result: [632b661](https://github.com/cerner/terra-clinical/pull/870/commits/632b661f37653bb9f89d4c4ddb50cfb172613676)